testsuite/features/step_definitions/common_steps.rb

```diff
@@ -90,7 +90,13 @@
 end
 
 When(/^I check the MAC address value$/) do
-  get_mac_address = 'cat /sys/class/net/eth0/address'
+  get_system_id_like = "awk -F= '/^ID_LIKE/{print $2}' /etc/os-release"
```

Comment:
Makes no sense, `$client` is hardcoded below, so it cannot be a Ubuntu minion.

Add a host parameter and do as usually.
```
   I check the MAC address value on "([^"]*)"$/) do |host|
  node = get_target(host)
  if host == 'ubuntu-minion'
```
then replace `$client` below with `node` so this step becomes truly generic.

Reply:
true